css/properties/align-items.json

```diff
@@ -2,305 +2,357 @@
-                "version_added": null
+                "version_added": "12.1"
               },
               "safari": {
-                "version_added": null
+                "prefix": "-webkit-",
+                "version_added": "7"
               },
               "safari_ios": {
-                "version_added": null
+                "prefix": "-webkit-",
+                "version_added": "7"
               },
               "samsunginternet_android": {
                 "version_added": null
-              }
+              },
+              "webview_android": [
+                {
+                  "version_added": "52"
+                },
+                {
+                  "version_added": "29",
+                  "partial_implementation": true,
+                  "notes": "Older versions of the specification treat absolute positioned children as though they are a 0 by 0 flex item. Later specification versions take the children out of the flow and set their positions based on align and justify properties. Chrome implements the new behavior beginning with Chrome 52."
+                },
+                {
+                  "prefix": "-webkit-",
+                  "version_added": "21"
+                }
+              ]
             },
             "status": {
               "experimental": false,
               "standard_track": true,
               "deprecated": false
             }
-          }
-        },
-        "start_end": {
-          "__compat": {
-            "description": "<code>start</code> and <code>end</code>",
-            "support": {
-              "webview_android": {
-                "version_added": null
-              },
-              "chrome": {
-                "version_added": null
-              },
-              "chrome_android": {
-                "version_added": null
-              },
-              "edge": {
-                "version_added": null
-              },
-              "edge_mobile": {
-                "version_added": null
-              },
-              "firefox": {
-                "version_added": "45"
-              },
-              "firefox_android": {
-                "version_added": "45"
-              },
-              "ie": {
-                "version_added": null
-              },
-              "opera": {
-                "version_added": null
-              },
-              "opera_android": {
-                "version_added": null
-              },
-              "safari": {
-                "version_added": null
-              },
-              "safari_ios": {
-                "version_added": null
-              },
-              "samsunginternet_android": {
-                "version_added": null
+          },
+          "first_last_baseline": {
+            "__compat": {
+              "description": "<code>first baseline</code> and <code>last baseline</code>",
+              "support": {
+                "webview_android": {
+                  "version_added": null
```

Comment:
It would be really nice to change some of these `null` values to Yes/No even if the version number is not known.